src/converters/converters.h

```diff
@@ -95,19 +95,19 @@ template<class T>
 struct std_item
 {
     typedef typename T::value_type V;
-    static V& get(T const& x, int i)
+    static V& get(T & x, int i)

```

Comment:
for the changes in this file - i'm not sure if it is a good idea to omit the "const" keyword; i followed the example in the URL above - so that's basically what they do;  I think using non-const references could give more flexibility, but also opens more room for potential problems, so it would be less safe to do them; Please revert the changes in this file.

Reply:
Ok. Let me think how to update it in a better way